api/v1alpha1/oadp_types.go

```diff
@@ -30,6 +30,7 @@ const ReconcileCompleteMessage = "Reconcile complete"
 
 const OadpOperatorLabel = "openshift.io/oadp"
 const RegistryDeploymentLabel = "openshift.io/oadp-registry"
+const DataMoverDeploymentLabel = "openshift.io/oadp-data-mover"
```

Comment:
Might want to change this after we finish discussion at https://github.com/openshift/openshift-velero-plugin/pull/141/files#r891349632 about oadp labels.

Reply:
can change later. testing now. visiack.

Reply:
Thank you !